app/assets/javascripts/tooltip.js

```diff
@@ -0,0 +1,19 @@
+var showTooltips = function(options) {
+  this.init = function() {
+    this.settings = jQuery.extend({}, defaultOptions, options);
+
+    jQuery(this.settings.element)
+    .focusin(function () {
+      jQuery(this).parent().parent().addClass('tooltip-visible');
```

Comment:
Well, in the usual case, but what if the tooltip was a component that does not care whether or not it is attachment to a form element?

The tooltip *should be reusable*, which means it's logic should *not* be tied to the specific use case. I can't stress that enough.